scripts/update-search-index.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+set -e
+
+readonly BASH_COMMONS_DIR="/opt/gruntwork/bash-commons"
+
+if [[ ! -d "$BASH_COMMONS_DIR" ]]; then
+  echo "ERROR: this script requires that bash-commons is installed in $BASH_COMMONS_DIR. See https://github.com/gruntwork-io/bash-commons for more info."
+  exit 1
+fi
+
+source "$BASH_COMMONS_DIR/log.sh"
+source "$BASH_COMMONS_DIR/assert.sh"
+
+readonly ALGOLIA_APP_ID=7AWZHGNJE2
```

Comment:
I don't think it's a huge deal. This file is config for the indexing while the `default.yaml` is config that drives the querying. In practice - I don't think this will cause us much pain as we're unlikely to require changing this value.

It would be nicer to source this value from one spot - but I don't see a super simple way to parse `yaml` in `bash` - so trying to source this from there doesn't seem like it's worth it.